metadata/metadata.go

```diff
@@ -94,6 +94,7 @@ const (
 	StatusCode          = "status code"
 	Syscall             = "system calls"
 	Thread              = "threads"
+	Timestamp           = "timestamp"
```

Comment:
Does bosun have capabilities to alert properly on timestamps? Would we be better of reporting time remaining till expiration?